qcodes/math/field_vector.py

```diff
@@ -7,11 +12,11 @@
 
 class FieldVector(object):
     """
-    A convenient class to keep track of vectors representing physical fields. The
-    idea is that a vector instance stores a representation in Cartesian, spherical
-    and cylindrical coordinates. Giving either (x, y, z) values or (rho, phi, z)
-    values or (r, theta, phi) values at instantiation, we will calculate the other
-    representation immediately.
+    The idea is that a vector instance stores a representation in Cartesian,
```

Comment:
I think it's still good to have "A class representing X" in the docstring. But that may be a matter of individual preference.